### PR TITLE
Add announcement about Carlos

### PR DESCRIPTION
**Pull request by kelnera**
@osu-cass/cass-media This announcement needs to go up quickly. I'll look into what the merge conflicts are, but I'd still appreciate it if you guys could take a look and let me know if you see any issues.

### CODE REVIEW

**Comment by kelnera** [comment]
@osu-cass/cass-media @osu-cass/cass-web Yay! Merge conflicts resolved! Please take a look and let me know if you see any other issues.

**Comment by kelnera** [comment]
@ramereth It's possible those are just updates that already exist but needed to be added to my program, but there was a lot that happened to resolve some weird issues. @Kennric @athai do you guys know what it is?

**Comment by ramereth** [comment]
@kelnera we don't want to delete that submodule. That needs to stay there.

**Comment by kelnera** [comment]
@ramereth Okay, we're going to keep working on this.

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [submodule "dougfir-pelican-theme"]
 	path = dougfir-pelican-theme
-	url = https://github.com/osuosl/dougfir-pelican-theme
+	url = git@github.com:osuosl/dougfir-pelican-theme.git
```
Comment (ramereth) [inline .gitmodules]: This should remain with https instead of git.

**Comment by ramereth** [comment]
@kelnera can you please make carlos's picture a little smaller? Not sure why it needs to be this big.

File: content/pages/organization/staff.rst [hunk 2]
```diff
@@ -14,7 +14,7 @@ Dr. Carlos Jensen
 
 .. class:: no-breaks
 
-  **Director Emeritus**
+  **CASS Director**
```
Comment (ramereth) [inline content/pages/organization/staff.rst]: This should still be ``CASS Director Emeritus`` correct?

**Comment by ramereth** [comment]
@kelnera can you please remove the old picture that was larger? There's no reason why we need to keep it.

**Comment by kelnera** [comment]
@ramereth The pictures are exactly the same actually. Well, sort of. The first picture is the one I'd like to use, and the second one is cheating the system. Our pages are currently set up to a fixed width and that's why the picture was so big. Ideally, it would be better to use the original picture, rather than the one we adjusted in photoshop (See #108). If you feel strongly about not have duplicates, I can still remove it, though.

**Comment by kelnera** [comment]
@athai Now that we've sort of resolved the submodule issue, we've come full circle in that the announcement isn't showing up on the home page. Yay.

**Comment by kelnera** [comment]
@ramereth I just went ahead and removed it. If the issue gets resolved, I can always replace the picture. We'd like to merge this today so if you have any other suggestions, please let me know soon.

**Pull request merged**